fixtures/eurolite/led-kls-801.json

```diff
@@ -6,10 +6,15 @@
   "meta": {
     "authors": ["Flo Edelmann", "Felix Edelmann"],
     "createDate": "2017-01-20",
-    "lastModifyDate": "2018-08-24"
+    "lastModifyDate": "2024-11-23"
   },
   "links": {
-    "manual": ["http://s141963554.online.de/anleitung/Steinigke/81012467.pdf"],
+    "manual": [
+      "https://www.steinigke.de/download/42109851-Manual-60662-1.200-eurolite-led-kls-801-compact-light-set-de_en.pdf"
```

Comment:
Is this manual actually the same one as the previously linked one? I just quickly glanced over it and saw that the previous one has 72 pages and the new one has 32 pages, but that might just be one more translation. In that case, updating it to only English would be fine.